dev.js

```diff
@@ -3,12 +3,12 @@ const chokidar = require('chokidar')
 
 electron.start()
 
-chokidar.watch('main.js', {
-  ignored: '*/node_modules/*'
-})
-.on('change', () => electron.restart())
+chokidar
+  .watch('main.js')
+  .on('change', () => electron.restart())
 
-chokidar.watch('**/*.(js|json|html)', {
-  ignored: '*/node_modules/*'
-})
-.on('change', () => electron.reload())
+chokidar
+  .watch('**/*.(js|json|html)', {
+    ignored: '**/data/**'
```

Comment:
fix bug where saving meta was reloading the app when in dev mode. lol.

also it seems relevant to reload when node_modules changes. i also couldnt see a way to add multiple `ignores`. thoughts?